legalize.js

```diff
@@ -20,12 +20,19 @@
 (function (factory) {
     "use strict";
 
-    if (typeof define === 'function' && define.amd) {
-        define('Legalize', factory);
+    var legalize = factory();
+
+    // set globally for use with plain javascript
+    if (typeof window !== "undefined") {
+        window.Legalize = legalize;
     } else {
-        window.Legalize = factory();
+        this.Legalize = legalize;
     }
 
+    // define module for use with requireJs
+    if (typeof define === "function" && define.amd) {
+        define("Legalize", [], legalize);
```

Comment:
Beforehand define was called with factory (without braces, so just the function that returns the 'frozen object'). For some reason this caused errors in using the Legalize with requirejs. Handing over the frozen object directly fixes this.